AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -857,7 +857,7 @@ - (void)layoutSubviews
   _nextLayoutInvalidationStyle = ASCollectionViewInvalidationStyleNone;
   switch (invalidationStyle) {
     case ASCollectionViewInvalidationStyleWithAnimation:
-      if (!_superPerformingBatchUpdates) {
+      if (0 == _superBatchUpdateCount) {
```

Comment:
@Adlai-Holler I understand the value in this style, but for now let's match the prevailing style, which is == NO